examples/app/stock_applet/stock_data.py

```diff
@@ -0,0 +1,65 @@
+import os
+from six.moves import urllib
+import zipfile
+
+
+def extract_hosted_zip(data_url, save_dir, exclude_term=None):
+    """Downloads, then extracts a zip file."""
+
+    zip_name = os.path.join(save_dir, 'temp.zip')
+
+    # get the zip file
+    try:
+        zip_name, hdrs = urllib.request.urlretrieve(url=data_url, filename=zip_name)
+    except IOError, e:
```

Comment:
this is not py3 compat... you can use `except IOError:` or `except IOError as e:`